frontstage/views/register.py

```diff
@@ -104,22 +105,22 @@ def register_confirm_organisation_survey():
     if encrypted_enrolment_code:
         decrypted_enrolment_code = ons_env.crypt.decrypt(encrypted_enrolment_code.encode()).decode()
     else:
-        logger.error('Confirm organisation screen - Enrolment code not specified')
+        logger.error('"event" : "error", "message" : "Confirm organisation screen - Enrolment code not specified"')
```

Comment:
The event should be your message here. The .error let's you know that it's an error